packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_table_row.dart

```diff
@@ -59,11 +59,12 @@ class LoggingTableRow extends StatefulWidget {
 class _LoggingTableRowState extends State<LoggingTableRow> {
   @override
   Widget build(BuildContext context) {
+    final theme = Theme.of(context);
     final color = widget.isSelected
-        ? Colors.blueGrey
+        ? theme.colorScheme.selectedRowBackgroundColor
         : alternatingColorForIndex(
             widget.index,
-            Theme.of(context).colorScheme,
+            theme.colorScheme,
```

Comment:
nit: we can store the colorScheme in the local variable here instead of the theme to save a lookup